plugins/network-elements/nicira-nvp/src/com/cloud/network/resource/NiciraNvpResource.java

```diff
@@ -180,6 +182,17 @@ public Type getType() {
     public PingCommand getCurrentStatus(final long id) {
         try {
             ControlClusterStatus ccs = niciraNvpApi.getControlClusterStatus();
+            //------------------------------------------------GET API_PROVIDER MAJORITY VERSION
```

Comment:
if this block is wrapped by a comment, wouldn't it be better to refactor it into a new method? (giving the method a meaningful name that would render the comment redundant, say `getApiProviderMajorityVersion(...)`)